app.js

```diff
@@ -0,0 +1,19 @@
+const express = require('express')
+const app = express()
+
+app.use(express.json())
```

Comment:
you're not using this line yet, it's used for parsing incoming request bodies eg. in a POST request